devices/debian_wifi.py

```diff
@@ -3,10 +3,9 @@
 import pexpect
 from countrycode import countrycode
 from lib.wifi import wifi_client_stub
-import pdb
```

Comment:
Why are you doing this? This commit is a copy of the previous commit message yet it's only contents are this?